server/index.js

```diff
@@ -36,6 +36,13 @@ app.get('/healthcheck', (req, res) => {
   res.send('OK')
 })
 
+// Handle warmup requests in a bare-minimum fashion
+// Could also use this hook to start up middleware etc, but not worth it for our use case
+// https://cloud.google.com/appengine/docs/standard/configuring-warmup-requests?tab=node.js
+app.get('/_ah/warmup', (req, res) => {
+  res.send('OK')
```

Comment:
That's fair. Doing this actually did help me during dev since it's simple to tell that adding the route worked, but it's not needed for prod. I'll remove it before merging.

Reply:
Actually, omitting it appears to cause the connection to hang client-side. We could use `res.sendStatus(200)` instead, but I'd rather not just let it hang in case that makes Google's side act weird. Going to leave this in.